apps/api/src/app/layouts/usecases/set-default-layout/set-default-layout.use-case.ts

```diff
@@ -19,6 +21,7 @@ export class SetDefaultLayoutUseCase {
     try {
       if (defaultLayoutId) {
         await this.setIsDefaultForLayout(defaultLayoutId, command.environmentId, command.organizationId, false);
+        await this.createLayoutChangeForPreviousDefault(command, defaultLayoutId);
```

Comment:
As we make non default the default layout, we need to keep track of that change. Not totally happy as this could be easily overlooked in a change of logic. Worth discuss it.